### DISCUSSION TITLE
Multi-task learning and ONNX support

### DISCUSSION THREAD

**Discussion by ogencoglu** [StatMixedML/LightGBMLSS#27] [Ideas]
Thanks for the great work!

I have two questions:

1) Can we perform multi-task learning in a single training where one task is classification and the target variables are categorical (classes) and the other task is regression where the target values are continuous?

2) Does LightGBMLSS models support ONNX conversions?

**Comment by StatMixedML** [comment]
See https://github.com/StatMixedML/XGBoostLSS/discussions/46
